configuration_version_integration_test.go

```diff
@@ -331,3 +354,21 @@ func TestConfigurationVersions_Unmarshal(t *testing.T) {
 	assert.Equal(t, cv.StatusTimestamps.FinishedAt, finishedParsedTime)
 	assert.Equal(t, cv.StatusTimestamps.StartedAt, startedParsedTime)
 }
+
+// helper to wait until a configuration version has reached a certain status
+func WaitUntilStatus(t *testing.T, client *Client, ctx context.Context, cv *ConfigurationVersion, desiredStatus ConfigurationStatus) {
```

Comment:
💞 I like this refactor and I love how the tests read now! 

I'm still new to this repo, so I'm going to ping @brandonc and @sebasslash because I'm not sure what we typically do with test helper methods. In this case, I'm not sure it makes sense to put it in helper_test.go, but that seems like the pattern we have used in the past. 🤔